### DISCUSSION TITLE
Looking for hardcore settings

### DISCUSSION THREAD

**Discussion by isid0re** [isid0re/SoloLeveling#58] [Q&A] [answered]
I've not used the bot with hardcore, but I would like to setup the config to use different settings for hardcore.

So far the settings I have are:
Lifechicken: 40
Usehp: 90
Userejuvhp: 65
Usemp: 45

Do these settings look right to you hardcore players? If not, can you provide a suggestion and explain why so I understand hardcore a little more. Thanks

Is it ok to use the same settings across the board for characters or should each character have different settings? 

If different, could you provide settings for the various classes? Thanks. 

Appreciate the dialogue and help.

**Comment by ebner20** [comment, marked as answer]
Life chicken should be 60.  I ran on a 40 and 50 setting before and got one shotted cuz they hit more than what i had for life at 50% setting.  I find the 60% good because I've seen it chicken with 15hp left after a big hit.  

Your potion settings are what I exactly use.

Dont forget to lower ping to around 255 @ 2 seconds.  This is a lifesaver (literally) when you have bad internet like me.  

Is there a setting you can input to select if your running hardcore vs. Softcore so it auto inputs settings?  Maybe have it run in nightmare until I has a high enough resist for hell?  We start at -100 so even making the bot farm gear so resists are back to 0 or higher would be great for us hardcore players.

**Reply by isid0re** [reply]
Great. I will update to 60. Ping settings are really user specific.

Yes, the way to tell in the config if it is hardcore or not is me.playertype. if its hardcore it returns true. 

I used to do the resists check before I implemented cows and full baal runs. It might be worthwhile going back to that. Especially for hardcore.

**Reply by ebner20** [reply]
Yeah I would agree.  I'm seriously considering creating a gui that will be super user friendly with simple radio buttons for different settings.  like you'll start off picking softcore mode or hardcore mode, what class, then what build.  each selection will then write into each js file required to make it work.  I'm tired of forgetting to change a setting or leaving one actives  i.e. doing runs and forgetting to turn off Anderial.  This will have a trimmed down visual of whatever user has selected.  

Kinda like whomever built the gui for the pickets.  I think its called Nipper?  It would be quite similar but I'm guessing I'm going to be taking on a huge task.  In my brain it says it can be simplified.  We will see.

**Reply by isid0re** [reply]
That's a cool idea. I'm not a programmer nor do I work in IT, so I had zero javascript experience when I started this back in april of last year. Google, reading the stock scripts while watching the script run are what taught me.

SoloLeveling started as a covid project for a smiter to source my own torch, but it kinda grew out from there. 😁

**Comment by ciompek** [comment]
Question is - how to add some customization for the bot to do a desired sequence only when reached a desired level?

**Reply by isid0re** [reply]
You will need to modify the completedtasks function.
